charts/zwavejs2mqtt/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.0.0-alpha.2
 description: Fully configurable Zwave to MQTT Gateway and Control Panel
 name: zwavejs2mqtt
-version: 1.0.0
+version: 1.0.1
```

Since the default value for the privileged mode changes, I would bump this to 1.1.0.
```suggestion
version: 1.1.0
```